File: design-patterns/go/composite/transport_list/plane.go
// plane.go

package main

import "fmt"

type Plane struct {
}

func NewPlane() (plane *Plane) {
	plane = &Plane{}
	return
}

func (plane *Plane) Start() {
	fmt.Println("Starting Plane...")
}

func (plane *Plane) Stop() {
	fmt.Println("Stopping Plane...")
}

func (plane *Plane) Operate() {
	fmt.Println("Flying Plane")
}

File: design-patterns/php/composite/transportlist/Plane.php
<?php
// Plane.php

namespace BigBoxCode\DesignPattern\Composite\TransportList;

class Plane implements Transport {
    public function start(): void {
        echo "Starting Plane...\n";
    }

    public function operate(): void {
        echo "Flying Plane\n";
    }

    public function stop(): void {
        echo "Stopping Plane...\n";
    }
}